File: Java/Program/Product.java
class Product{
    private String idProduct, name, brand, price; //String private untuk beberapa string id, name, brand, price

    public Product() { //Konstruktor default untuk product
        this.idProduct = "";
        this.name = "";
        this.brand = "";
        this.price = "";
    }
    //Konstruktor parameter untuk product
    public Product(String idProduct, String name, String brand, String price ){
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }
    //Method untuk set idProduct
    public void setIdProduct(String idProduct) {
        this.idProduct = idProduct;
    }
    //Method untuk set brand
    public void setBrand(String brand) {
        this.brand = brand;
    }
    //Method untuk set name
    public void setName(String name) {
        this.name = name;
    }
    //method untuk set price
    public void setPrice(String price) {
        this.price = price;
    }
    //Method untuk get brand 
    public String getBrand() {
        return brand;
    }
    //Method untuk get idProduct
    public String getIdProduct() {
        return idProduct;
    }
    //method untuk get name
    public String getName() {
        return name;
    }
    //Method untuk get price
    public String getPrice() {
        return price;
    }
}

File: CPP/Program/Product.cpp
#include <string>
using namespace std;

class Product {
private:
    string idProduct, name, brand, price; // String private untuk beberapa string id, name, brand, price

public:
    Product() { // Konstruktor default untuk product
        idProduct = "";
        name = "";
        brand = "";
        price = "";
    }

    // Method untuk set idProduct
    void setIdProduct(string idProduct) {
        this->idProduct = idProduct;
    }

    // Method untuk set brand
    void setBrand(string brand) {
        this->brand = brand;
    }

    // Method untuk set name
    void setName(string name) {
        this->name = name;
    }

    // method untuk set price
    void setPrice(string price) {
        this->price = price;
    }

    // Method untuk get brand
    string getBrand() {
        return brand;
    }

    // Method untuk get idProduct
    string getIdProduct() {
        return idProduct;
    }

    // method untuk get name
    string getName() {
        return name;
    }

    // Method untuk get price
    string getPrice() {
        return price;
    }

    ~Product(){
    }
};